settings.xml

```diff
@@ -101,12 +101,41 @@
       </pluginRepositories>
     </profile>
 
+    <!-- Configure the JBoss Tech Preview Maven repository -->
```

Comment:
This is not needed, GA repository should be used instead.

Reply:
@nziakova : The GA repository is listed first. Both the "early access" and "tech preview" repositories are there in case they are ever needed. We have needed them in the past. Should I move the tech preview out of the active profiles section or remove it entirely?

Reply:
techpreview repo has been replaced by the GA one, so it's not needed any more

Reply:
earlyaccess is used for Alpha/Beta releases, ga is used for GA releases

Reply:
I have removed tech preview.